js/commerce_pos.transaction.js

```diff
@@ -23,6 +23,16 @@
         }
       });
 
+      // Show ajax throbber while we're searching.
+      $('.commerce-pos-product-search').keyup(function() {
+        if ($(this).val() != '') {
+          $(this).addClass('show-ajax-throbber');
+        }
+        else {
+          $(this).removeClass('show-ajax-throbber');
+        }
+      });
+
```

Comment:
Can we use the form api's ajax throbber for this? https://api.drupal.org/api/drupal/developer%21topics%21forms_api_reference.html/7.x#ajax_progress